docs/tutorials/ruby-driver-create-client.txt

```diff
@@ -97,8 +97,23 @@ URI Options Conversions
    * - URI Option
      - Ruby Option
 
-   * - replicaSet=String
-     - ``:replica_set => String``
+   * - appName=String
+     - ``:app_name => String``
+
+   * - authMechanism=String
+     - ``:auth_mech => Symbol``
+
+   * - authMechanismProperties=Strings
+     - ``{ :auth_mech_properties => { :service_realm => String, :canonicalize_host_name => true|false, :service_name => String } }``
+
+       Specified as comma-separated key:value pairs, e.g. ``"SERVICE_REALM:foo,CANONICALIZE_HOSTNAME:TRUE"``
+
+   * - authSource=String
+     - ``:auth_source => String``
+
+   * - compressors=Strings
+     - ``:compressors => Array<String>``
+       Specified as a comma-separated list, e.g. "zlib,snappy"
```

Comment:
This is a weird default to specify in the Ruby driver because snappy isn't supported by Ruby. Will this warn?

Perhaps a better language is something along the lines of:

> Specified as a comma-separated list. Note that the Ruby driver only supports zlib compression, however other drivers may support snappy. For maximum compatibility with drivers specify "zlib,snappy", if compatibility with other drivers is not a concern specify "zlib".

Reply:
I didn't intend this to be interpreted as a default, but rather just to demonstrate how specifying more than one would work. I'm happy to make the suggested change, though.

Reply:
Okay that makes sense, let's put this language in given user confusion over compression.

Reply:
Just pushed